android/expoview/src/main/java/host/exp/exponent/LoadingView.java

```diff
@@ -138,19 +142,25 @@ public void setManifest(JSONObject manifest) {
       mImageView.setVisibility(View.GONE);
       final String iconUrl = loadingInfo.optString(ExponentManifest.MANIFEST_LOADING_ICON_URL);
       mIsLoadingImageView = true;
-      Picasso.with(getContext()).load(iconUrl).into(mImageView, new Callback() {
-        @Override
-        public void onSuccess() {
-          revealView(mImageView);
-          mIsLoadingImageView = false;
-          AsyncCondition.notify(ASYNC_CONDITION_KEY);
-        }
+      Glide
+          .with(getContext())
+          .load(iconUrl)
+          .listener(new RequestListener<Drawable>() {
+            @Override
+            public boolean onLoadFailed(@Nullable GlideException e, Object model, Target<Drawable> target, boolean isFirstResource) {
```

Comment:
Can we make one function that loads Glide so we don't have duplicated code here?